src/main/scala/com/ing/wbaa/airlock/proxy/api/ProxyService.scala

```diff
@@ -41,15 +41,17 @@ trait ProxyService extends LazyLogging {
 
   val proxyServiceRoute: Route =
     withoutSizeLimit {
-      extractClientIP { clientIPAddress =>
+      (extractClientIP & extractForwardedForIPs) { (clientIPAddress, forwardedForAddresses) =>
```

Comment:
we have to check **X-Real-IP** header as well

Reply:
done! now we are matching the IPs of the TCP connection IP (this was already implemented) and the headers `Remote-Address`, `X-Real-IP`, `X-Forwarded-For`